script/Ciel/Day 12/12_01_CIEL12_3_2A.txt

```diff
@@ -51,27 +51,28 @@ He'll understand immediately if he hears that I couldn't make it due to poor hea
 [sha:aba3ff228d8bb38d85e43e3b1e68b6993738304f]{
 -- Page 183, Offset 31356.
 -- 乾さまにご連絡した後、お薬を持って行きますから。どうかお部屋のベッドでお休みください」
-After contacting Inui-sama, I'll prepare some medicine and bring them to you. So please lie on your bed and get some rest."
+After contacting Inui-sama, I'll bring you some medicine, so please lie on your bed and get some rest."
 }
 [sha:a55ede54db116b956f7bf901141f15f2bcd9df34]{
 -- Page 184, Offset 31357.
 -- ……彼女は着物の<裾|すそ>をわずかに乱しながら、小走りで東館に向かって行った。
-...Disfiguring the hems of her kimono slightly, Kohaku-san quickly makes her way toward the east wing.
+// i dont know how the fuck kimonos work. hem or hems?
+...Disheveling the hem of her kimono slightly, Kohaku-san quickly makes her way toward the east wing.
```

Comment:
"disheveling" and "disfiguring" are both kinda wrong. "Messing up the bottom of her kimono slightly,"
![image](https://user-images.githubusercontent.com/16092757/184685823-c7ecd929-ea86-4613-a2db-635870cbf700.png)